src/Service/UserTransformer.php

```diff
@@ -128,6 +131,7 @@ public function changePasswordTransform(
             throw $passwordDoNotMatch;
         }
         $user->setPassword($this->encoder->encodePassword($user, $dto->password));
```

Comment:
Hint: inside `setPassword()` you could use `$this->setPasswordChangedAt(new DateTime('now'));` and it will always update the timestamp when the password will be changed.